pipeline/pipeline.go

```diff
@@ -606,7 +608,7 @@ func (p *Pipeline) expandProcs() {
 	}
 
 	for x := 0; x < int(to-from); x++ {
-		proc := p.newProc(p.Procs[from-1].id + x)
```

Comment:
can you add comment about this ? why it is important here and what we are doing

Reply:
done